lib/sinja/sequel/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Sinja
   module Sequel
-    VERSION = '0.1.6'
+    VERSION = '0.1.7'
```

Comment:
I'll bump the version number when I do a release. You can take this change out. Thank you!